server/src/client.js

```diff
@@ -101,72 +101,16 @@ class Client {
       return this.close({ error: 'Invalid handshake.', error_code: 0 });
     }
 
-    const success = (user_feed, token) => {
```

Comment:
@Tryneus wasn't sure about the `user_feed` stuff here and following and why we need it.